protos/v0/jvs_plugin_service.proto

```diff
@@ -25,6 +25,8 @@ option go_package = "github.com/abcxyz/jvs/apis/v0";
 service JVSPlugin {
   rpc Validate(ValidateJustificationRequest)
       returns (ValidateJustificationResponse);
+  rpc GetUiData(GetUiDataRequest)
```

Comment:
Sanity check, is it backward compatible when providing a new rpc without default implementation? My guess is this will break the existing jira plugin since the existing code no longer implement this new interface. If that is the case, may want to research some way to make backward compatible changes

Reply:
I will update the [plug ](https://github.com/abcxyz/jvs-plugin-jira) shortly after merging this change.

Reply:
Note that you will need to update both of the following places
- jvs-plugin-jira repo
- https://github.com/abcxyz/jvs/blob/main/apis/v0/plugin.go

Reply:
Oh I see your point. Maybe we should update https://github.com/abcxyz/jvs/blob/main/apis/v0/plugin.go in this PR (or right after this one)

Reply:
Thank you! I will update https://github.com/abcxyz/jvs/blob/main/apis/v0/plugin.go in the next one.